fdbserver/ClusterController.actor.cpp

```diff
@@ -5348,12 +5358,39 @@ ACTOR Future<Void> startBlobManager(ClusterControllerData* self) {
 	}
 }
 
+ACTOR Future<Void> watchBlobGranulesConfigKey(ClusterControllerData* self) {
+	state Reference<ReadYourWritesTransaction> tr = makeReference<ReadYourWritesTransaction>(self->cx);
+	state Key blobGranuleConfigKey = configKeysPrefix.withSuffix(StringRef("blob_granules_enabled"));
```

Comment:
```suggestion
	state Key blobGranuleConfigKey = configKeysPrefix.withSuffix("blob_granules_enabled"_sr);
```

Reply:
You want to use LiteralStringRef() for constructing a string ref from c-style strings. "abc"_sr is shorthand for LiteralStringRef("abc").
The build failed because of this warning

Reply:
Ah, thanks for pointing that out!